packages/runtime/container-runtime-definitions/src/containerRuntime.ts

```diff
@@ -59,9 +65,18 @@ export interface IContainerRuntime
 	 * Returns the runtime of the data store.
 	 * @param id - Id supplied during creating the data store.
 	 * @param wait - True if you want to wait for it.
+	 * @deprecated - Use getAliasedDataStore instead to get an aliased data store.
 	 */
 	getRootDataStore(id: string, wait?: boolean): Promise<IFluidRouter>;
 
+	/**
+	 * Returns an aliased data store with the given alias.
```

Comment:
If you take my suggestions on the class comment, remember to copy them here too.